lib/ReactViews/Map/Panels/HelpPanel/HelpPanelItem.jsx

```diff
@@ -102,8 +102,16 @@ class HelpPanelItem extends React.Component {
           itemString={this.props.content.itemName}
           paneMode={this.props.content.paneMode}
           markdownContent={this.props.content.markdownText}
-          videoUrl={this.props.content.videoUrl}
-          placeholderImage={this.props.content.placeholderImage}
+          videoUrl={
+            this.props.content.videoUrl
+              ? t(this.props.content.videoUrl)
+              : this.props.content.videoUrl
```

Comment:
I think it would make sense to use `i18n.exists(this.props.content.videoUrl)` here like in`HelpVideoPanel.jsx` for example. Or is there a reason it's not used in this case? I'm just thinking for cases where `this.props.content.videoUrl` exists but the translation of it doesn't

Reply:
Oh wait I'm an idiot I just realised this is a link/url. In which case do we need translations for them?

Reply:
This enables users to provide a different help video made for different languages (although there could be cc provide for video). 

`i18next.exists` is needed in two cases (if I recall correctly):
- when loading an objects from translation (arrays)
- when dealing with larger text that can get into translation function. Default behavior of `t` function should be to put a translation text if there is no translation, but when there are larger or quiet complex text it happens to be trimmed, so we use `i18next.exists` to check if that string exists in any of translation files.
 
just noticed a few places that we can add `i18next.exists` to avoid second issue , `markdown` in `StyledHtml`.

Reply:
Ah that's fair thanks for clarifying!

Reply:
Yes that's right, I've addressed & simplified this in https://github.com/TerriaJS/terriajs/pull/4613/commits/434f49af1c5b3f7f1833b572a4789da8ee597ae2 & https://github.com/TerriaJS/terriajs/pull/4613/commits/880900543647021301246e164540db8ed06e3c77 now

Reply:
& actually more importantly doc update so this info isn't lost to just this PR https://github.com/TerriaJS/terriajs/pull/4613/commits/782bde028ee82ac33b69db615d61ff4ed26ae2a5